lib/server_sent_event.ex

```diff
@@ -188,9 +252,11 @@ defmodule ServerSentEvent do
   defp process_line(line, event) do
     case String.split(line, ~r/: ?/, parts: 2) do
       ["", value] ->
-        process_field("comment", value, event)
+        {:ok, process_field("comment", value, event)}
```

Comment:
`process_field` should return :ok/:error. It would error in the case of an invalid field name